raiden-ts/src/path/epics.ts

```diff
@@ -61,37 +61,28 @@ const oneToNAddress = memoize(
 
 /**
  * Codec for PFS API returned error
+ *
+ * May contain other fields like error_details, but we don't care about them (for now)
  */
 const PathError = t.readonly(
```

Comment:
Thanks for removing I was thinking to remove them at some point and then I completely forgot it :)

Reply:
This wouldn't usually be a problem, but I've stumbled upon `error_details` which didn't match the provided schema. Seems it's error-dependent, so either we kept it as `t.any` or just remove it, as we don't inspect this property.

Reply:
Removing is better than having dead code. I was thinking that before merging and then I missed it so thanks for taking care of it :)